Untitled.py

```diff
@@ -0,0 +1,53 @@
+
+
+
+def sheet_data(sheet):
+
+    out = list()
+
+    sheetIndex = 0
+    for X in range(1, s.nrows):
+        learner = s.row_values(X)
+        print(learner)
+        average=sum(learner)
+        out.append([learner[1], average])
+
+    return out
+
+def sum(learner):
+    average = 0
+    for score in range(2, len(learner)) :
+        average = average+learner[score] // 2
```

Comment:
If we want to get average, we should divide on number of marks (11).

Reply:
So your recommendation is to replace `2` with `11`?

Reply:
I think, it's better to remove 2 in that line at all and rewrite line 23 as something like `return average//(len(learner)-2)` to make it a bit more universal

Reply:
Not sure about the divisor, but agree with the general idea.